tests/infra/AutocompleteServiceImpl.ts

```diff
@@ -33,6 +34,18 @@ describe('AutocompleteService', () => {
 
   const mockHttpService = new HttpServiceMock();
   const apiResponseValidator = new ApiResponseValidator();
+  function createMockAutocompleteService(params?: { response?: unknown, answersConfig?: AnswersConfig }) {
```

Comment:
can the type be some union of the vertical and universal autocomplete response rather than unknown?

Reply:
oh hmm this is the raw api response so I don't think so. I considered making it
```ts
mockAutocompleteResponse | mockAutocompleteResponseWithSections | //etc
```
but didn't really see much value in that

Reply:
oh I see, so can it be of type ApiResponse?

Reply:
ah yeah that's better!